certora/Makefile

```
@@ -3,9 +3,9 @@ default: help
 SRC   := ../contracts
 DST   := patched
 DIFF  := diff
-SRCS  := $(shell find $(SRC) -type f)
-DSTS  := $(shell find $(DST) -type f)
-DIFFS := $(shell find $(DIFF) -type f)
+SRCS  := $(sh find $(SRC) -type f)
+DSTS  := $(sh find $(DST) -type f)
+DIFFS := $(sh find $(DIFF) -type f)
```

https://www.gnu.org/software/make/manual/html_node/Shell-Function.html

```suggestion
SRCS  := $(shell find $(SRC) -type f)
DSTS  := $(shell find $(DST) -type f)
DIFFS := $(shell find $(DIFF) -type f)
```

Did you run into an error with the original code?

---

I did. I think this is equivalent and doesn't require extra configuration from my side.
I can do it on local if needed.

---

Can you check what implementation and version of make you have? Is it GNU make?

---

Yes it is.

```shell
$ make -v
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
```

What doesn't seem to work is the find though:

```shell
$ make -C certora apply
find: diff: No such file or directory
```

---

I see, so that's unrelated, I've seen the same message in the logs for the workflow. Let's look at that in another PR.